src/service-disruptions/ServiceDisruptionSheet.tsx

```diff
@@ -33,7 +33,7 @@ export const ServiceDisruptionSheet = forwardRef<View>(({}, focusRef) => {
         {hasValidServiceDisruptionUrl && (
           <>
             <Section style={style.serviceDisruptionText}>
-              <View ref={focusRef} accessible>
+              <View>
                 <ThemeText>{t(ServiceDisruptionsTexts.body)}</ThemeText>
               </View>
```

Comment:
This `<View>` looks redundant now
```suggestion
              <ThemeText>{t(ServiceDisruptionsTexts.body)}</ThemeText>
```